package/test-package.sh

```diff
@@ -28,7 +28,7 @@ kevm run tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json -
     || git --no-pager diff --no-index --ignore-all-space -R tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.llvm-out tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.expected
 rm -rf tests/failing/static_callcodecallcodecall_110_OOGMAfter_2_d0g0v0.json.llvm-out
 
-kevm solc-to-k tests/specs/examples/ERC20.sol ERC20 > tests/specs/examples/erc20-bin-runtime.k
+kevm_pyk solc-to-k /usr/bin/lib/haskell/driver-kompiled tests/specs/examples/ERC20.sol ERC20 > tests/specs/examples/erc20-bin-runtime.k
```

Comment:
We should still do `kevm solc-to-k ...` instead, so that this argument is not necessary.

The whole point of the `kevm` script is to "know where it's installed" and do any environment setup needed to make other scripts work with fewer arguments.

So this added argument is unecessary.